docs/monitoring/reports-and-components.md

```diff
@@ -1,62 +1,68 @@
 
-### Split Screen
+![Sematext Cloud Monitoring Guide - Group By](../images/monitoring/group-by-infra.png)
 
-[Split Screen](../guide/split-screen) gives you a single view for Metrics, Logs, and Events. This is a huge time saver to let you move from **Metrics spikes** to **drill down to Logs and Events** that caused them.
+Another example is the grouping of the [infrastructure storage charts](https://apps.sematext.com/ui/infrastructure/serversStorage) by the `os.disk` tag, allowing us to quickly check for I/O spikes in specific disks.
 
-![Sematext Cloud Monitoring Guide - Monitoring Split Screen](../images/guide/monitoring/monitoring-split-screen.png) 
+![Sematext Cloud Monitoring Guide - Group By](../images/monitoring/group-by-osdisk.png)
 
-Combined with the **Metric Reports**, and with the help of a **charting hairline**, zoom and chart navigation tools, it is easy to see how using our **Monitoring and Logging tools in a single page** is a must-have for any DevOps team! Learn more about **[Split Screen](/guide/split-screen)**.
+Similar customized group menus can be created using [`Group By` report variables](https://sematext.com/docs/dashboards/report-variables/#groups).
 
-The set of icons in the logs search and report menu let you:
+### Add Alert
 
-- **Save as View** - Filtering and grouping data is a central part of the Sematext Cloud user interface. With Saved Views you can save your filters and groups for faster access later. If you find yourself often applying the same filters on your data then this feature can be a time saver.
-- **Add Alert** - Every query can be use as an **alert query**. The ability to save alerts is part of the essential tools for searching, correlating and investigating logs data..
+Every query can be use as an **alert query**. The ability to save alerts is part of the essential tools for searching, correlating and investigating logs data..
 Each alert query is visible in the Alert Rules list.
 
-- **Scheduled Reports** - There are E-mail and Subscription types available with time, frequency, and time range defined so your team can get automatated reports about your logs and system health. Learn more about **[Scheduled Reports](/guide/scheduled-reports)**.
+### Scheduled Reports
 
-- **Full Screen Mode** - Entering full screen mode brings only the component you choose into view. It's useful for presentations and viewing detailed info of a single component. All charts have zoom, crosshair, and time navigation tools with tooltips available for effective chart interaction and data analysis.
+There are E-mail and Subscription types available with time, frequency, and time range defined so your team can get automatated reports about your logs and system health. Learn more about **[Scheduled Reports](/guide/scheduled-reports)**.
```

Comment:
Subscription types?  Is this referring to Scheduled Reports? (we used to call them Subscriptions)

Reply:
Nice name btw :)